README.md

```diff
@@ -262,6 +262,12 @@ Capability | Description
 |`appium:clearSystemFiles`|Whether to clean temporary XCTest files (for example logs) when a testing session is closed. `false` by default| `true` or `false`
 |`appium:newCommandTimeout`|How long (in seconds) the driver should wait for a new command from the client before assuming the client has stopped sending requests. After the timeout the session is going to be deleted. `60` seconds by default. Setting it to zero disables the timer. |e.g. `100`|
 
+### Settings
```

Comment:
I would rather add this information to the main appium doc and have a link in each driver doc. Eventually, this is not the main way settings should be used.

Reply:
there is the stub of a settings API doc which we can fill out. ultimately the work that @boneskull will lead to auto-generation of specific settings docs for extensions, so we will be able to extract this information from the codebase and simply refer users to the settings API doc for a conceptual understanding of the API and some examples. then they can use the individual driver docs as a reference for the particular settings available for each driver.